### DISCUSSION TITLE
Table with filter

### DISCUSSION THREAD

**Discussion by Tomlennon21** [nhsevidence/NICE-Experience#154] [General]
I was looking to see if we could have the ability to create a table with a filter. 

This has come about through discussions on the [cost saving guidance](https://www.nice.org.uk/about/what-we-do/into-practice/cost-saving-guidance) page. They were looking at being able to enter keywords such as "bed days" or "theatres" and for that then to filter out the relevant report for them. The problem we may have depends on the amount of categories and the tagging involved. We couldn't filter by words in the title as the info would be inside an excel or pdf. We also need something that only filters upon pressing enter for accessibility reasons.

**Comment by wa-rren-dev** [comment]
If it wasn't 'tagged' as such and there was just free-form text matching then could that work? 

An option for tags in a still-reusable way might be to have a hidden table column that's also matched.

**Comment by Tomlennon21** [comment]
As long as we could link a certain word, say "strokes" to a particular document then that would be ideal. 

We were also discussing about a possible search facility in a web page along the same idea. Mainly to be used for searching for a particular copy on a page with a lot of content.
